west.yml

```diff
@@ -98,7 +98,8 @@ manifest:
       revision: 1c4fdba512b268033a4cf926bddd323866c3261a
       path: tools/net-tools
     - name: hal_nxp
-      revision: 478f03a71256aeb2890e685ebf5a1dc6832a6b5f
+      # revision: 478f03a71256aeb2890e685ebf5a1dc6832a6b5f
+      revision: pull/47/head
```

Comment:
The hal driver has been merged, so you can now update this to the new SHA

Reply:
ok